runtime/runtime.py

```diff
@@ -523,6 +575,7 @@ def send_motor_data(data):
     })
 
     # Update sensor values, and send to UI
+    update_hibike()
```

Comment:
I could try going to 101 tomorrow to test/tweak the code I already wrote to see if I could get it to work.